page_documentation.py

```diff
@@ -8,102 +8,82 @@
 [Matthijs van der Wild](https://github.com/lonbar)
 """
 
-import numpy as np
 import streamlit as st
-from scipy import integrate
-import background as bg
-import matplotlib.pyplot as plt
-import plot_script as ps
+from helpers import read_markdown, read_eq
 
 def show_page():
-    st.sidebar.write("")
-    st.sidebar.write("")
-    st.sidebar.write("")
-    st.sidebar.write("")
-    st.sidebar.write("")
-    st.sidebar.write("")
-    st.sidebar.write("")
-    st.sidebar.write("")
-
-    st.title('Definitions of quantities')
-    st.markdown('This page collates the definitions and meaning of the quantities that are computed by CosmΩracle.')
-    
-    st.markdown('Details can be found in *Distance Measures in Cosmology* by D. Hogg, [astro-ph/9905116](https://arxiv.org/abs/astro-ph/9905116).')
-
-    st.header("Parameters")
 
-    st.subheader("Redshift z")
-    st.markdown("The redshift _z_ is the increase in wavelength of electromagnetic radiation as a result of the increase in the cosmological scale factor _a_. It can be computed in terms of the scale factor via the relation")
-    st.latex(r'''1 + z = \frac{1}{a},''')
-    st.markdown("where it is conventional to take the present value of the scale factor to be equal to 1.")
+    for i in range(8):
+        st.sidebar.write("")
 
-    st.subheader("Hubble constant H₀")
-    st.markdown("The Hubble parameter _H₀_ quantifies the relative expansion of space. It is computed from the scale factor _a_ and its time derivative via the relation")
-    st.latex(r'''H = \frac{\dot{a}}{a}.''')
-    st.markdown("The parameter _H₀_ parametrises the value of the Hubble parameter in the current epoch.")
+    definitions = read_markdown("docs/markdown/definitions.md")
+    st.markdown(definitions)
+    
 
-    st.subheader("Energy density")
-    st.markdown("The evolution of the scale factor is driven by the energy density of the matter content of the Universe. The dimensionless matter density Ω<sub>M</sub>, photon density Ω<sub>R</sub> and dark energy density Ω<sub>DE</sub> parametrise energy contents of baryonic matter, radiation and dark energy, respectively. A curvature density parameter Ω<sub>K</sub> is defined through the relation", unsafe_allow_html=True)
-    st.latex(r'''\Omega_{\rm M} + \Omega_{\rm R} + \Omega_{\rm DE} + \Omega_{\rm K} = 1.''')
-    st.markdown("Ω<sub>K</sub> characterises the curvature of space: if Ω<sub>K</sub> < 0, then the Universe has a finite spatial extent. If Ω<sub>K</sub> = 0, then the Universe is spatially flat. If Ω<sub>K</sub> > 0, then the Universe is spatially curved, but infinite in extent.", unsafe_allow_html=True)
+    st.header("Parameters")
```

Comment:
To be honest, I wasn't quite convinced by this choice. However now, looking at the document which is much shorter, I am VERY convinced.